src/brogue/Rogue.h

```diff
@@ -168,6 +168,9 @@ typedef struct pos {
 
 #define AMULET_LEVEL            26          // how deep before the amulet appears
 #define DEEPEST_LEVEL           40          // how deep the universe goes
+#define DEPTH_ACCELERATOR       1           // factor for how fast depth-dependent features scale compared to usual 26-level dungeon
```

Comment:
This is probably OK to ensure that these changes don't break replays, but it feels a bit nicer to me if we were to rewrite the algorithms (e.g. the light one) to use a % of depth as a parameter. You can see that sort of thing being done for example in Architect.c:2392

```
const short descentPercent = clamp(100 * (rogue.depthLevel - 1) / (AMULET_LEVEL - 1), 0, 100);
```